packages/osc-ui/src/components/Avatar/Avatar.tsx

```diff
@@ -21,9 +23,16 @@ export const Avatar: FC<AvatarProps> = (props: AvatarProps) => {
         : null;
 
     return (
-        <AvatarPrimitive.Root className={classes}>
+        <AvatarPrimitive.Root className={src ? classes : `${classes} c-avatar__fallback`}>
             <AvatarPrimitive.Image className="c-avatar__image" src={src} alt={name} />
-            <AvatarPrimitive.Fallback className="c-avatar__fallback">
+            {notification.show ? (
+                <div className="c-avatar__notification-badge">
```

Comment:
try to keep class names as short and concise as possible (i can appreciate that this is not always the case) so for this example i would suggest:

```suggestion
                <div className="c-avatar__badge">
```

To coincide with this, i would image we would utilise the badge component here?
`<Badge />`

Reply:
Updated classnames and utilised Badge component 👍